charts/controller/templates/controller-ingress-rule-controller.yaml

```diff
@@ -0,0 +1,21 @@
+{{ if .Values.global.experimental_native_ingress }}
```

Comment:
I've made a similar comment on some other PRs related to this same feature.

I think it's clear to me that we're going to have to explore some more creative patterns for how, in the "parent" chart, one flag can be set that enables this feature and have that setting can _propagate_ down to dependent charts like the router chart and controller chart. This might be easier said than done, but we're going to have to find a way.